src/NUnitFramework/tests/Api/DefaultTestAssemblyBuilderTests.cs

```diff
@@ -24,32 +24,13 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Reflection;
 using NUnit.Framework.Internal;
 using NUnit.Tests;
 using NUnit.Tests.Assemblies;
 using NUnit.Tests.Singletons;
-using NUnit.Compatibility;
-using NUnit.TestData;
 
 namespace NUnit.Framework.Api
 {
-    [TestFixture] // [TestFixture] is needed so that ClassesCanBeImpliedToBeFixtures is effective.
```

Comment:
@CharliePoole I think it's important to preserve this comment on whatever class contains `ClassesCanBeImpliedToBeFixtures`.

Reply:
I'll add it if you like. However, I believe we have other of tests that would not work correctly without specifying `[TestFixture]`. So far, we have just done it without comment, just as we avoid using any of the features we are testing in the tests themselves.

Reply:
If this is generally known by the team (I missed it) then it makes sense not to mention it. The cost of adding it doesn't seem great, unless it would further cause us to assume (as I already did) that all other `[TestFixture]` attributes could be removed.